diff --git a/configuration/dv_application.tf b/configuration/dv_application.tf
new file mode 100644
index 0000000..fb5c085
--- /dev/null
+++ b/configuration/dv_application.tf
@@ -0,0 +1,27 @@
+resource "davinci_application" "registration_flow_app" {
+  name           = "DaVinci API Protect Sample Application"
+  environment_id = pingone_group_role_assignment.terraform_sso_davinci_admin.scope_environment_id
+
+  oauth {
+    enabled = true
+    values {
+      allowed_grants                = ["authorizationCode"]
+      allowed_scopes                = ["openid", "profile"]
+      enabled                       = true
+      enforce_signed_request_openid = false
+      redirect_uris                 = ["${module.pingone_utils.pingone_url_auth_path_full}/rp/callback/openid_connect"]
+    }
+  }
+}
+
+resource "davinci_application_flow_policy" "registration_flow_app_policy" {
+  environment_id = pingone_group_role_assignment.terraform_sso_davinci_admin.scope_environment_id
+  application_id = davinci_application.registration_flow_app.id
+  name           = "DaVinci API Protect Sample Policy"
+  status         = "enabled"
+  policy_flow {
+    flow_id    = davinci_flow.pingone_davinci_api_protect_example.id
+    version_id = -1
+    weight     = 100
+  }
+}
\ No newline at end of file
diff --git a/configuration/dv_connection.tf b/configuration/dv_connection.tf
new file mode 100644
index 0000000..63abe77
--- /dev/null
+++ b/configuration/dv_connection.tf
@@ -0,0 +1,79 @@
+// Flow Name: PingOne DaVinci API Protect Example
+resource "davinci_connection" "httpconnector__867ed4363b2bc21c860085ad2baa817d" {
+  environment_id = pingone_group_role_assignment.terraform_sso_davinci_admin.scope_environment_id
+
+  connector_id = "httpConnector"
+  name         = "Http"
+}
+
+// Flow Name: PingOne DaVinci API Protect Example
+resource "davinci_connection" "pingoneriskconnector__292873d5ceea806d81373ed0341b5c88" {
+  environment_id = pingone_group_role_assignment.terraform_sso_davinci_admin.scope_environment_id
+
+  connector_id = "pingOneRiskConnector"
+  name         = "PingOne Protect"
+
+  property {
+    name  = "clientId"
+    type  = "string"
+    value = pingone_application.worker_app.oidc_options.client_id
+  }
+
+  property {
+    name  = "clientSecret"
+    type  = "string"
+    value = pingone_application_secret.worker_app_secret.secret
+  }
+
+  property {
+    name  = "envId"
+    type  = "string"
+    value = pingone_group_role_assignment.terraform_sso_davinci_admin.scope_environment_id
+  }
+
+  property {
+    name  = "region"
+    type  = "string"
+    value = var.pingone_client_region_code
+  }
+}
+
+// Flow Name: PingOne DaVinci API Protect Example
+resource "davinci_connection" "pingonessoconnector__94141bf2f1b9b59a5f5365ff135e02bb" {
+  environment_id = pingone_group_role_assignment.terraform_sso_davinci_admin.scope_environment_id
+
+  connector_id = "pingOneSSOConnector"
+  name         = "PingOne"
+
+  property {
+    name  = "clientId"
+    type  = "string"
+    value = pingone_application.worker_app.oidc_options.client_id
+  }
+
+  property {
+    name  = "clientSecret"
+    type  = "string"
+    value = pingone_application_secret.worker_app_secret.secret
+  }
+
+  property {
+    name  = "envId"
+    type  = "string"
+    value = pingone_group_role_assignment.terraform_sso_davinci_admin.scope_environment_id
+  }
+
+  property {
+    name  = "region"
+    type  = "string"
+    value = var.pingone_client_region_code
+  }
+}
+
+// Flow Name: PingOne DaVinci API Protect Example
+resource "davinci_connection" "variablesconnector__06922a684039827499bdbdd97f49827b" {
+  environment_id = pingone_group_role_assignment.terraform_sso_davinci_admin.scope_environment_id
+
+  connector_id = "variablesConnector"
+  name         = "Variables"
+}
\ No newline at end of file
diff --git a/configuration/dv_flow.tf b/configuration/dv_flow.tf
new file mode 100644
index 0000000..05d43ea
--- /dev/null
+++ b/configuration/dv_flow.tf
@@ -0,0 +1,50 @@
+##########################################################################
+# davinci.tf - Declarations to create DaVinci assets
+# {@link https://registry.terraform.io/providers/pingidentity/davinci/latest}
+##########################################################################
+
+#########################################################################
+# PingOne DaVinci - Create and deploy a flow
+#########################################################################
+# {@link https://registry.terraform.io/providers/pingidentity/davinci/latest/docs/resources/flow}
+
+// Flow Name: PingOne DaVinci API Protect Example
+resource "davinci_flow" "pingone_davinci_api_protect_example" {
+
+  environment_id = pingone_group_role_assignment.terraform_sso_davinci_admin.scope_environment_id
+
+  name        = "PingOne DaVinci API Protect Example"
+  description = "This flow demonstrates how to protect an app with PingOne"
+
+  flow_json = file("./davinci_flows/davinci-api-protect-reg-authn-flow.json")
+
+  log_level = 2
+
+  // Connector link: httpConnector
+  connection_link {
+    id                           = davinci_connection.httpconnector__867ed4363b2bc21c860085ad2baa817d.id
+    name                         = davinci_connection.httpconnector__867ed4363b2bc21c860085ad2baa817d.name
+    replace_import_connection_id = "867ed4363b2bc21c860085ad2baa817d"
+  }
+
+  // Connector link: pingOneRiskConnector
+  connection_link {
+    id                           = davinci_connection.pingoneriskconnector__292873d5ceea806d81373ed0341b5c88.id
+    name                         = davinci_connection.pingoneriskconnector__292873d5ceea806d81373ed0341b5c88.name
+    replace_import_connection_id = "292873d5ceea806d81373ed0341b5c88"
+  }
+
+  // Connector link: pingOneSSOConnector
+  connection_link {
+    id                           = davinci_connection.pingonessoconnector__94141bf2f1b9b59a5f5365ff135e02bb.id
+    name                         = davinci_connection.pingonessoconnector__94141bf2f1b9b59a5f5365ff135e02bb.name
+    replace_import_connection_id = "94141bf2f1b9b59a5f5365ff135e02bb"
+  }
+
+  // Connector link: variablesConnector
+  connection_link {
+    id                           = davinci_connection.variablesconnector__06922a684039827499bdbdd97f49827b.id
+    name                         = davinci_connection.variablesconnector__06922a684039827499bdbdd97f49827b.name
+    replace_import_connection_id = "06922a684039827499bdbdd97f49827b"
+  }
+}
\ No newline at end of file
diff --git a/configuration/local_file_env.tf b/configuration/local_file_env.tf
index 2f8e94f..82d3620 100644
--- a/configuration/local_file_env.tf
+++ b/configuration/local_file_env.tf
@@ -8,7 +8,7 @@ resource "local_file" "env_config_simple_p1_reg_app" {
 }
 
 resource "local_file" "env_config_dv_p1_protect_reg_app" {
-  content  = "window._env_ = {\n  pingOneDomain: \"${module.pingone_utils.pingone_domain_suffix}\",\n  pingOneEnvId: \"${pingone_environment.target_environment.id}\",\n  clientId: \"${pingone_application.client_auth_sample_app.id}\", \n companyId: \"${module.ping-export.dv_app_environment_id}\", \n apiKey: \"${module.ping-export.dv_app_api_key}\",\n  policyId: \"${module.ping-export.dv_policy_id}\"\n};"
+  content  = "window._env_ = {\n  pingOneDomain: \"${module.pingone_utils.pingone_domain_suffix}\",\n  pingOneEnvId: \"${pingone_environment.target_environment.id}\",\n  clientId: \"${pingone_application.client_auth_sample_app.id}\", \n companyId: \"${davinci_application.registration_flow_app.environment_id}\", \n apiKey: \"${davinci_application.registration_flow_app.api_keys.prod}\",\n  policyId: \"${davinci_application_flow_policy.registration_flow_app_policy.id}\"\n};"
   filename = "../dv-p1-protect-reg-app/global.js"
 }
 
diff --git a/configuration/module.tf b/configuration/module.tf
deleted file mode 100644
index f9c29b9..0000000
--- a/configuration/module.tf
+++ /dev/null
@@ -1,16 +0,0 @@
-module "ping-export" {
-  source = "./ping-export-module"
-
-  pingone_environment_id = pingone_environment.target_environment.id
-
-  # Connection Variables
-  davinci_connection_PingOne-0020-Protect_clientId     = pingone_application.worker_app.oidc_options.client_id
-  davinci_connection_PingOne-0020-Protect_clientSecret = pingone_application_secret.worker_app_secret.secret
-  davinci_connection_PingOne-0020-Protect_envId        = pingone_application.worker_app.environment_id
-  davinci_connection_PingOne-0020-Protect_region       = var.pingone_client_region_code
-  davinci_connection_PingOne_envId                     = pingone_application.worker_app.environment_id
-  davinci_connection_PingOne_region                    = var.pingone_client_region_code
-  davinci_connection_PingOne_clientId                  = pingone_application.worker_app.oidc_options.client_id
-  davinci_connection_PingOne_clientSecret              = pingone_application_secret.worker_app_secret.secret
-
-}
diff --git a/configuration/ping-export-module/outputs.tf b/configuration/ping-export-module/outputs.tf
deleted file mode 100644
index a89c1ea..0000000
--- a/configuration/ping-export-module/outputs.tf
+++ /dev/null
@@ -1,15 +0,0 @@
-output "dv_app_environment_id" {
-  description = "Environment ID for the DaVinci sample application"
-  value       = pingone_davinci_application.pingcli__DaVinci-0020-API-0020-Protect-0020-Sample-0020-Application.environment_id
-}
-
-output "dv_app_api_key" {
-  description = "API key value for the DaVinci sample application"
-  value       = pingone_davinci_application.pingcli__DaVinci-0020-API-0020-Protect-0020-Sample-0020-Application.api_key.value
-  sensitive   = true
-}
-
-output "dv_policy_id" {
-  description = "Flow policy ID for the DaVinci sample application"
-  value       = pingone_davinci_application_flow_policy.pingcli__DaVinci-0020-API-0020-Protect-0020-Sample-0020-Policy.id
-}
diff --git a/configuration/ping-export-module/pingone_davinci_application.tf b/configuration/ping-export-module/pingone_davinci_application.tf
deleted file mode 100644
index f8d6d0c..0000000
--- a/configuration/ping-export-module/pingone_davinci_application.tf
+++ /dev/null
@@ -1,16 +0,0 @@
-
-resource "pingone_davinci_application" "pingcli__DaVinci-0020-API-0020-Protect-0020-Sample-0020-Application" {
-  environment_id = var.pingone_environment_id
-
-  name = "DaVinci API Protect Sample Application"
-
-  api_key = {
-    enabled = true
-  }
-
-  oauth = {
-    grant_types   = ["authorizationCode"]
-    redirect_uris = ["https://auth.pingone.com/9804c259-1f4f-4fda-b0a8-3f3b7339d5ce/rp/callback/openid_connect"]
-    scopes        = ["openid", "profile"]
-  }
-}
\ No newline at end of file
diff --git a/configuration/ping-export-module/pingone_davinci_application_flow_policy.tf b/configuration/ping-export-module/pingone_davinci_application_flow_policy.tf
deleted file mode 100644
index 8274a50..0000000
--- a/configuration/ping-export-module/pingone_davinci_application_flow_policy.tf
+++ /dev/null
@@ -1,16 +0,0 @@
-# Flow Policies (1 total)
-
-resource "pingone_davinci_application_flow_policy" "pingcli__DaVinci-0020-API-0020-Protect-0020-Sample-0020-Policy" {
-  environment_id         = var.pingone_environment_id
-  davinci_application_id = pingone_davinci_application.pingcli__DaVinci-0020-API-0020-Protect-0020-Sample-0020-Application.id
-  name                   = "DaVinci API Protect Sample Policy"
-  status                 = "enabled"
-
-  flow_distributions = [
-    {
-      id      = pingone_davinci_flow.pingcli__PingOne-0020-DaVinci-0020-API-0020-Protect-0020-Example.id
-      version = -1
-      weight  = 100
-    },
-  ]
-}
\ No newline at end of file
diff --git a/configuration/ping-export-module/pingone_davinci_connector_instance.tf b/configuration/ping-export-module/pingone_davinci_connector_instance.tf
deleted file mode 100644
index 241ba83..0000000
--- a/configuration/ping-export-module/pingone_davinci_connector_instance.tf
+++ /dev/null
@@ -1,81 +0,0 @@
-
-
-resource "pingone_davinci_connector_instance" "pingcli__Http" {
-  environment_id = var.pingone_environment_id
-
-  name = "Http"
-
-  connector = {
-    id = "httpConnector"
-  }
-}
-
-
-resource "pingone_davinci_connector_instance" "pingcli__PingOne" {
-  environment_id = var.pingone_environment_id
-
-  name = "PingOne"
-
-  connector = {
-    id = "pingOneSSOConnector"
-  }
-
-  properties = jsonencode({
-    "clientId" : {
-      "type" : "string",
-      "value" : "${var.davinci_connection_PingOne_clientId}"
-    },
-    "clientSecret" : {
-      "type" : "string",
-      "value" : "${var.davinci_connection_PingOne_clientSecret}"
-    },
-    "envId" : {
-      "type" : "string",
-      "value" : "${var.davinci_connection_PingOne_envId}"
-    },
-    "region" : {
-      "type" : "string",
-      "value" : "${var.davinci_connection_PingOne_region}"
-    }
-  })
-}
-
-
-resource "pingone_davinci_connector_instance" "pingcli__PingOne-0020-Protect" {
-  environment_id = var.pingone_environment_id
-
-  name = "PingOne Protect"
-
-  connector = {
-    id = "pingOneRiskConnector"
-  }
-
-  properties = jsonencode({
-    "clientId" : {
-      "type" : "string",
-      "value" : "${var.davinci_connection_PingOne-0020-Protect_clientId}"
-    },
-    "clientSecret" : {
-      "type" : "string",
-      "value" : "${var.davinci_connection_PingOne-0020-Protect_clientSecret}"
-    },
-    "envId" : {
-      "type" : "string",
-      "value" : "${var.davinci_connection_PingOne-0020-Protect_envId}"
-    },
-    "region" : {
-      "type" : "string",
-      "value" : "${var.davinci_connection_PingOne-0020-Protect_region}"
-    }
-  })
-}
-
-resource "pingone_davinci_connector_instance" "pingcli__Variables" {
-  environment_id = var.pingone_environment_id
-
-  name = "Variables"
-
-  connector = {
-    id = "variablesConnector"
-  }
-}
\ No newline at end of file
diff --git a/configuration/ping-export-module/pingone_davinci_flow.tf b/configuration/ping-export-module/pingone_davinci_flow.tf
deleted file mode 100644
index ab300b2..0000000
--- a/configuration/ping-export-module/pingone_davinci_flow.tf
+++ /dev/null
@@ -1,1105 +0,0 @@
-
-resource "pingone_davinci_flow" "pingcli__PingOne-0020-DaVinci-0020-API-0020-Protect-0020-Example" {
-  environment_id = var.pingone_environment_id
-
-  name        = "PingOne DaVinci API Protect Example"
-  description = "This flow demonstrates how to protect an app with PingOne"
-  color       = "#CACED3"
-
-  settings = {
-    csp                              = "worker-src 'self' blob:; script-src 'self' https://cdn.jsdelivr.net https://code.jquery.com https://devsdk.singularkey.com http://cdnjs.cloudflare.com 'unsafe-inline' 'unsafe-eval';"
-    intermediate_loading_screen_css  = ""
-    intermediate_loading_screen_html = ""
-    log_level                        = 2
-  }
-
-  graph_data = {
-    elements = {
-      nodes = {
-        "09anefv002" = {
-          data = {
-            id        = "09anefv002"
-            node_type = "EVAL"
-            properties = jsonencode({
-              "7h8bs4vxuo" = {
-                "value" = "allTriggersFalse"
-              },
-              "u6w4ob97dt" = {
-                "value" = "allTriggersFalse"
-              }
-            })
-          }
-          position = {
-            x = 1357
-            y = 326
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "0mqv2pv49t" = {
-          data = {
-            id              = "0mqv2pv49t"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__Variables.id
-            connector_id    = "variablesConnector"
-            name            = "Variables"
-            label           = "Variables"
-            status          = "configured"
-            capability_name = "saveValue"
-            type            = "trigger"
-            properties = jsonencode({
-              "saveVariables" = {
-                "value" = [
-                  {
-                    "key"         = 0.5428342378678097,
-                    "label"       = "userId (string - flowInstance)",
-                    "name"        = "userId",
-                    "nameDefault" = "id",
-                    "type"        = "string",
-                    "value"       = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"pingIdentity.svg\",\n        \"url\": \"id\",\n        \"data\": \"{{local.6i7lwwrw94.payload.output.user.id}}\",\n        \"tooltip\": \"{{local.6i7lwwrw94.payload.output.user.id}}\",\n        \"children\": [\n          {\n            \"text\": \"id\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-                  }
-                ]
-              }
-            })
-          }
-          position = {
-            x = 877
-            y = 476
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "1jf93tyrvi" = {
-          data = {
-            id              = "1jf93tyrvi"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__Variables.id
-            connector_id    = "variablesConnector"
-            name            = "Variables"
-            label           = "Variables"
-            status          = "configured"
-            capability_name = "saveValue"
-            type            = "trigger"
-            properties = jsonencode({
-              "saveVariables" = {
-                "value" = [
-                  {
-                    "key"         = 0.8779096515773732,
-                    "label"       = "userId (string - flowInstance)",
-                    "name"        = "userId",
-                    "nameDefault" = "id",
-                    "type"        = "string",
-                    "value"       = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"pingIdentity.svg\",\n        \"url\": \"id\",\n        \"data\": \"{{local.m4sfmek769.payload.output.matchedUser.id}}\",\n        \"tooltip\": \"{{local.m4sfmek769.payload.output.matchedUser.id}}\",\n        \"children\": [\n          {\n            \"text\": \"id\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-                  }
-                ]
-              }
-            })
-          }
-          position = {
-            x = 877
-            y = 236
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "6i7lwwrw94" = {
-          data = {
-            id              = "6i7lwwrw94"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__PingOne.id
-            connector_id    = "pingOneSSOConnector"
-            name            = "PingOne"
-            label           = "PingOne"
-            status          = "configured"
-            capability_name = "createUser"
-            type            = "action"
-            properties = jsonencode({
-              "additionalUserProperties" = {
-                "value" = [
-                ]
-              },
-              "email" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"email\",\n        \"data\": \"{{global.parameters.email}}\",\n        \"tooltip\": \"{{global.parameters.email}}\",\n        \"children\": [\n          {\n            \"text\": \"email\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              },
-              "passwordForCreateUser" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"password\",\n        \"data\": \"{{global.parameters.password}}\",\n        \"tooltip\": \"{{global.parameters.password}}\",\n        \"children\": [\n          {\n            \"text\": \"password\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              },
-              "population" = {
-                "value" = "useDefaultPopulation"
-              },
-              "username" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"email\",\n        \"data\": \"{{global.parameters.email}}\",\n        \"tooltip\": \"{{global.parameters.email}}\",\n        \"children\": [\n          {\n            \"text\": \"email\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              }
-            })
-          }
-          position = {
-            x = 577
-            y = 476
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "75k6wc3j9i" = {
-          data = {
-            id        = "75k6wc3j9i"
-            node_type = "EVAL"
-            properties = jsonencode({
-              "jr1f287nlq" = {
-                "value" = "allTriggersFalse"
-              }
-            })
-          }
-          position = {
-            x = 697
-            y = 266
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "87jn430mui" = {
-          data = {
-            id        = "87jn430mui"
-            node_type = "EVAL"
-            properties = jsonencode({
-              "kc7wrmpc93" = {
-                "value" = "allTriggersFalse"
-              }
-            })
-          }
-          position = {
-            x = 727
-            y = 476
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "a6rirjd0m2" = {
-          data = {
-            id        = "a6rirjd0m2"
-            node_type = "EVAL"
-          }
-          position = {
-            x = 1072
-            y = 401
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "c9u3f1nyw7" = {
-          data = {
-            id              = "c9u3f1nyw7"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__Http.id
-            connector_id    = "httpConnector"
-            name            = "Http"
-            label           = "Http"
-            status          = "configured"
-            capability_name = "createSuccessResponse"
-            type            = "action"
-            properties = jsonencode({
-              "claimsNameValuePairs" = {
-                "value" = [
-                  {
-                    "key"         = 0.3076393052734512,
-                    "name"        = null,
-                    "nameDefault" = "level",
-                    "type"        = "string",
-                    "value"       = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"pingIdentity.svg\",\n        \"url\": \"level\",\n        \"data\": \"{{local.l9q5mxjjtm.payload.output.rawResponse.result.level}}\",\n        \"tooltip\": \"{{local.l9q5mxjjtm.payload.output.rawResponse.result.level}}\",\n        \"children\": [\n          {\n            \"text\": \"level\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-                  },
-                  {
-                    "key"         = 0.18337199639822654,
-                    "name"        = null,
-                    "nameDefault" = "score",
-                    "type"        = "string",
-                    "value"       = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"pingIdentity.svg\",\n        \"url\": \"score\",\n        \"data\": \"{{local.l9q5mxjjtm.payload.output.rawResponse.result.score}}\",\n        \"tooltip\": \"{{local.l9q5mxjjtm.payload.output.rawResponse.result.score}}\",\n        \"children\": [\n          {\n            \"text\": \"score\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-                  },
-                  {
-                    "key"         = 0.5430462947447765,
-                    "name"        = null,
-                    "nameDefault" = "rawResponse",
-                    "type"        = "string",
-                    "value"       = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"pingIdentity.svg\",\n        \"url\": \"rawResponse\",\n        \"data\": \"{{local.l9q5mxjjtm.payload.output.rawResponse}}\",\n        \"tooltip\": \"{{local.l9q5mxjjtm.payload.output.rawResponse}}\",\n        \"children\": [\n          {\n            \"text\": \"rawResponse\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-                  },
-                  {
-                    "key"         = 0.10159057010255945,
-                    "label"       = null,
-                    "name"        = null,
-                    "nameDefault" = "success",
-                    "type"        = null,
-                    "value"       = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"pingIdentity.svg\",\n        \"url\": \"success\",\n        \"data\": \"{{local.6i7lwwrw94.payload.success}}\",\n        \"tooltip\": \"{{local.6i7lwwrw94.payload.success}}\",\n        \"children\": [\n          {\n            \"text\": \"success\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-                  }
-                ]
-              },
-              "returnRequestParameters" = {
-                "value" = true
-              }
-            })
-          }
-          position = {
-            x = 1860
-            y = 270
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "gznrygimzf" = {
-          data = {
-            id              = "gznrygimzf"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__Http.id
-            connector_id    = "httpConnector"
-            name            = "Http"
-            label           = "Http"
-            status          = "configured"
-            capability_name = "createErrorResponse"
-            type            = "action"
-            properties = jsonencode({
-              "backgroundColor" = {
-                "value" = "#d0021bff"
-              },
-              "claimsNameValuePairs" = {
-                "value" = [
-                  {
-                    "key"         = 0.8800805675019314,
-                    "name"        = null,
-                    "nameDefault" = "message",
-                    "type"        = "string",
-                    "value"       = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"pingIdentity.svg\",\n        \"url\": \"message\",\n        \"data\": \"{{local.sxdpclcyko.payload.error.message}}\",\n        \"tooltip\": \"{{local.sxdpclcyko.payload.error.message}}\",\n        \"children\": [\n          {\n            \"text\": \"message\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-                  }
-                ]
-              }
-            })
-          }
-          position = {
-            x = 1860
-            y = 420
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "jr1f287nlq" = {
-          data = {
-            id              = "jr1f287nlq"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__Http.id
-            connector_id    = "httpConnector"
-            name            = "Http"
-            label           = "Http"
-            status          = "configured"
-            capability_name = "createErrorResponse"
-            type            = "action"
-            properties = jsonencode({
-              "backgroundColor" = {
-                "value" = "#d0021bff"
-              },
-              "claimsNameValuePairs" = {
-                "value" = [
-                  {
-                    "key"         = 0.617681003488906,
-                    "name"        = null,
-                    "nameDefault" = "message",
-                    "type"        = "string",
-                    "value"       = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"pingIdentity.svg\",\n        \"url\": \"message\",\n        \"data\": \"{{local.p53gq7n410.payload.error.message}}\",\n        \"tooltip\": \"{{local.p53gq7n410.payload.error.message}}\",\n        \"children\": [\n          {\n            \"text\": \"message\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-                  }
-                ]
-              }
-            })
-          }
-          position = {
-            x = 877
-            y = 386
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "kc7wrmpc93" = {
-          data = {
-            id              = "kc7wrmpc93"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__Http.id
-            connector_id    = "httpConnector"
-            name            = "Http"
-            label           = "Http"
-            status          = "configured"
-            capability_name = "createErrorResponse"
-            type            = "action"
-            properties = jsonencode({
-              "backgroundColor" = {
-                "value" = "#d0021bff"
-              },
-              "claimsNameValuePairs" = {
-                "value" = [
-                  {
-                    "key"         = 0.578323540163898,
-                    "name"        = null,
-                    "nameDefault" = "message",
-                    "type"        = "string",
-                    "value"       = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"pingIdentity.svg\",\n        \"url\": \"message\",\n        \"data\": \"{{local.6i7lwwrw94.payload.error.message}}\",\n        \"tooltip\": \"{{local.6i7lwwrw94.payload.error.message}}\",\n        \"children\": [\n          {\n            \"text\": \"message\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-                  }
-                ]
-              }
-            })
-          }
-          position = {
-            x = 877
-            y = 596
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "l9q5mxjjtm" = {
-          data = {
-            id              = "l9q5mxjjtm"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__PingOne-0020-Protect.id
-            connector_id    = "pingOneRiskConnector"
-            name            = "PingOne Protect"
-            label           = "PingOne Protect"
-            status          = "configured"
-            capability_name = "createRiskEvaluation"
-            type            = "action"
-            properties = jsonencode({
-              "additionalUserProperties" = {
-                "value" = [
-                ]
-              },
-              "cookie" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"cookies\",\n        \"data\": \"{{global.cookies}}\",\n        \"tooltip\": \"{{global.cookies}}\",\n        \"children\": [\n          {\n            \"text\": \"cookies\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              },
-              "ipAddress" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"ip\",\n        \"data\": \"{{global.ip}}\",\n        \"tooltip\": \"{{global.ip}}\",\n        \"children\": [\n          {\n            \"text\": \"ip\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              },
-              "password" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"password\",\n        \"data\": \"{{global.parameters.password}}\",\n        \"tooltip\": \"{{global.parameters.password}}\",\n        \"children\": [\n          {\n            \"text\": \"password\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              },
-              "skRiskFP" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"riskData\",\n        \"data\": \"{{global.parameters.riskData}}\",\n        \"tooltip\": \"{{global.parameters.riskData}}\",\n        \"children\": [\n          {\n            \"text\": \"riskData\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              },
-              "targetResourceName" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"DaVinci API Protect Sample App\"\n      }\n    ]\n  }\n]"
-              },
-              "userAgent" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"userAgent\",\n        \"data\": \"{{global.userAgent}}\",\n        \"tooltip\": \"{{global.userAgent}}\",\n        \"children\": [\n          {\n            \"text\": \"userAgent\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              },
-              "userId" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"variable.svg\",\n        \"url\": \"userId\",\n        \"data\": \"{{global.variables.userId}}\",\n        \"tooltip\": \"{{global.variables.userId}}\",\n        \"children\": [\n          {\n            \"text\": \"userId\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              },
-              "userName" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"email\",\n        \"data\": \"{{global.parameters.email}}\",\n        \"tooltip\": \"{{global.parameters.email}}\",\n        \"children\": [\n          {\n            \"text\": \"email\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              },
-              "userType" = {
-                "value" = "PING_ONE"
-              }
-            })
-          }
-          position = {
-            x = 1237
-            y = 326
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "m4sfmek769" = {
-          data = {
-            id              = "m4sfmek769"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__PingOne.id
-            connector_id    = "pingOneSSOConnector"
-            name            = "PingOne"
-            label           = "PingOne"
-            status          = "configured"
-            capability_name = "userLookup"
-            type            = "action"
-            properties = jsonencode({
-              "matchAttributes" = {
-                "value" = [
-                  "email"
-                ]
-              },
-              "userIdentifierForFindUser" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"email\",\n        \"data\": \"{{global.parameters.email}}\",\n        \"tooltip\": \"{{global.parameters.email}}\",\n        \"children\": [\n          {\n            \"text\": \"email\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              }
-            })
-          }
-          position = {
-            x = 277
-            y = 266
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "n6js2rcdqf" = {
-          data = {
-            id        = "n6js2rcdqf"
-            node_type = "EVAL"
-            properties = jsonencode({
-              "gznrygimzf" = {
-                "value" = "allTriggersFalse"
-              }
-            })
-          }
-          position = {
-            x = 1695
-            y = 270
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "p53gq7n410" = {
-          data = {
-            id              = "p53gq7n410"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__PingOne.id
-            connector_id    = "pingOneSSOConnector"
-            name            = "PingOne"
-            label           = "PingOne"
-            status          = "configured"
-            capability_name = "checkPassword"
-            type            = "action"
-            properties = jsonencode({
-              "identifier" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"email\",\n        \"data\": \"{{global.parameters.email}}\",\n        \"tooltip\": \"{{global.parameters.email}}\",\n        \"children\": [\n          {\n            \"text\": \"email\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              },
-              "matchAttribute" = {
-                "value" = "email"
-              },
-              "password" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"auth.svg\",\n        \"url\": \"password\",\n        \"data\": \"{{global.parameters.password}}\",\n        \"tooltip\": \"{{global.parameters.password}}\",\n        \"children\": [\n          {\n            \"text\": \"password\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              }
-            })
-          }
-          position = {
-            x = 577
-            y = 266
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "rvgypb5svi" = {
-          data = {
-            id        = "rvgypb5svi"
-            node_type = "EVAL"
-          }
-          position = {
-            x = 1072
-            y = 281
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "sxdpclcyko" = {
-          data = {
-            id              = "sxdpclcyko"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__PingOne-0020-Protect.id
-            connector_id    = "pingOneRiskConnector"
-            name            = "PingOne Protect"
-            label           = "PingOne Protect"
-            status          = "configured"
-            capability_name = "updateRiskEvaluation"
-            type            = "action"
-            properties = jsonencode({
-              "completionStatus" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"SUCCESS\"\n      }\n    ]\n  }\n]"
-              },
-              "riskId" = {
-                "value" = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"pingIdentity.svg\",\n        \"url\": \"id\",\n        \"data\": \"{{local.l9q5mxjjtm.payload.output.rawResponse.id}}\",\n        \"tooltip\": \"{{local.l9q5mxjjtm.payload.output.rawResponse.id}}\",\n        \"children\": [\n          {\n            \"text\": \"id\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-              }
-            })
-          }
-          position = {
-            x = 1560
-            y = 270
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "u6w4ob97dt" = {
-          data = {
-            id              = "u6w4ob97dt"
-            node_type       = "CONNECTION"
-            connection_id   = pingone_davinci_connector_instance.pingcli__Http.id
-            connector_id    = "httpConnector"
-            name            = "Http"
-            label           = "Http"
-            status          = "configured"
-            capability_name = "createErrorResponse"
-            type            = "action"
-            properties = jsonencode({
-              "backgroundColor" = {
-                "value" = "#d0021bff"
-              },
-              "claimsNameValuePairs" = {
-                "value" = [
-                  {
-                    "key"         = 0.7318766750020025,
-                    "name"        = null,
-                    "nameDefault" = "message",
-                    "type"        = "string",
-                    "value"       = "[\n  {\n    \"children\": [\n      {\n        \"text\": \"\"\n      },\n      {\n        \"text\": \"\"\n      },\n      {\n        \"type\": \"link\",\n        \"src\": \"pingIdentity.svg\",\n        \"url\": \"message\",\n        \"data\": \"{{local.l9q5mxjjtm.payload.error.message}}\",\n        \"tooltip\": \"{{local.l9q5mxjjtm.payload.error.message}}\",\n        \"children\": [\n          {\n            \"text\": \"message\"\n          }\n        ]\n      },\n      {\n        \"text\": \"\"\n      }\n    ]\n  }\n]"
-                  }
-                ]
-              }
-            })
-          }
-          position = {
-            x = 1560
-            y = 420
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-        "yqi3iaujxx" = {
-          data = {
-            id        = "yqi3iaujxx"
-            node_type = "EVAL"
-            label     = "Evaluator"
-            properties = jsonencode({
-              "0di26c5iy7" = {
-                "value" = "anyTriggersFalse"
-              },
-              "6i7lwwrw94" = {
-                "value" = "allTriggersFalse"
-              }
-            })
-          }
-          position = {
-            x = 427
-            y = 266
-          }
-          group      = "nodes"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = false
-          classes    = ""
-        }
-      }
-      edges = {
-        "05t56lofq2" = {
-          data = {
-            id     = "05t56lofq2"
-            source = "09anefv002"
-            target = "sxdpclcyko"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "06eo6sq6c8" = {
-          data = {
-            id     = "06eo6sq6c8"
-            source = "0mqv2pv49t"
-            target = "a6rirjd0m2"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "07z104pd2b" = {
-          data = {
-            id     = "07z104pd2b"
-            source = "75k6wc3j9i"
-            target = "1jf93tyrvi"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "53h4nrs4u0" = {
-          data = {
-            id     = "53h4nrs4u0"
-            source = "n6js2rcdqf"
-            target = "gznrygimzf"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "6n86yrphlt" = {
-          data = {
-            id     = "6n86yrphlt"
-            source = "a6rirjd0m2"
-            target = "l9q5mxjjtm"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "6uwzgd9xgy" = {
-          data = {
-            id     = "6uwzgd9xgy"
-            source = "1jf93tyrvi"
-            target = "rvgypb5svi"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "7mi39hz4x5" = {
-          data = {
-            id     = "7mi39hz4x5"
-            source = "87jn430mui"
-            target = "kc7wrmpc93"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "8cy8txv5dv" = {
-          data = {
-            id     = "8cy8txv5dv"
-            source = "l9q5mxjjtm"
-            target = "09anefv002"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "apual4vawu" = {
-          data = {
-            id     = "apual4vawu"
-            source = "m4sfmek769"
-            target = "yqi3iaujxx"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "cygeg7p11i" = {
-          data = {
-            id     = "cygeg7p11i"
-            source = "yqi3iaujxx"
-            target = "p53gq7n410"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "gjatfkco0t" = {
-          data = {
-            id     = "gjatfkco0t"
-            source = "09anefv002"
-            target = "u6w4ob97dt"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "jqobmjbxm9" = {
-          data = {
-            id     = "jqobmjbxm9"
-            source = "p53gq7n410"
-            target = "75k6wc3j9i"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "jsl67wfxc3" = {
-          data = {
-            id     = "jsl67wfxc3"
-            source = "75k6wc3j9i"
-            target = "jr1f287nlq"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "jwovpclfvp" = {
-          data = {
-            id     = "jwovpclfvp"
-            source = "rvgypb5svi"
-            target = "l9q5mxjjtm"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "poco6s1p07" = {
-          data = {
-            id     = "poco6s1p07"
-            source = "yqi3iaujxx"
-            target = "6i7lwwrw94"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "tr8r1rok8t" = {
-          data = {
-            id     = "tr8r1rok8t"
-            source = "6i7lwwrw94"
-            target = "87jn430mui"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "vsise0s8iw" = {
-          data = {
-            id     = "vsise0s8iw"
-            source = "87jn430mui"
-            target = "0mqv2pv49t"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "wv1og0m5r3" = {
-          data = {
-            id     = "wv1og0m5r3"
-            source = "sxdpclcyko"
-            target = "n6js2rcdqf"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-        "zwftxnuob4" = {
-          data = {
-            id     = "zwftxnuob4"
-            source = "n6js2rcdqf"
-            target = "c9u3f1nyw7"
-          }
-          position = {
-            x = 0
-            y = 0
-          }
-          group      = "edges"
-          removed    = false
-          selected   = false
-          selectable = true
-          locked     = false
-          grabbable  = true
-          pannable   = true
-          classes    = ""
-        }
-      }
-    }
-
-    pan = {
-      x = 0
-      y = 0
-    }
-
-    zoom                  = 1
-    min_zoom              = 1e-50
-    max_zoom              = 1e+50
-    zooming_enabled       = true
-    panning_enabled       = true
-    user_zooming_enabled  = true
-    user_panning_enabled  = true
-    box_selection_enabled = true
-
-    renderer = jsonencode({ "name" : "null" })
-  }
-
-  input_schema = [
-    {
-      property_name          = "email"
-      preferred_data_type    = "string"
-      preferred_control_type = "textField"
-      required               = true
-      is_expanded            = true
-      description            = ""
-    },
-    {
-      property_name          = "password"
-      preferred_data_type    = "string"
-      preferred_control_type = "textField"
-      required               = true
-      is_expanded            = true
-      description            = ""
-    },
-    {
-      property_name          = "riskData"
-      preferred_data_type    = "string"
-      preferred_control_type = "textField"
-      required               = true
-      is_expanded            = true
-      description            = ""
-    }
-  ]
-
-  lifecycle {
-    ignore_changes = [
-      connectors,
-      current_version,
-      enabled,
-      published_version
-    ]
-  }
-}
\ No newline at end of file
diff --git a/configuration/ping-export-module/variables.tf b/configuration/ping-export-module/variables.tf
deleted file mode 100644
index 7fcb28a..0000000
--- a/configuration/ping-export-module/variables.tf
+++ /dev/null
@@ -1,46 +0,0 @@
-variable "pingone_environment_id" {
-  type        = string
-  description = "The PingOne environment ID to configure DaVinci resources in"
-
-  validation {
-    condition     = can(regex("^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$", var.pingone_environment_id))
-    error_message = "The PingOne Environment ID must be a valid PingOne resource ID (UUID format)."
-  }
-}
-
-# Connection Variables
-
-variable "davinci_connection_PingOne-0020-Protect_clientId" {
-  type        = string
-  description = "clientId for PingOne Protect connector"
-}
-variable "davinci_connection_PingOne-0020-Protect_clientSecret" {
-  type        = string
-  description = "clientSecret for PingOne Protect connector"
-  sensitive   = true
-}
-variable "davinci_connection_PingOne-0020-Protect_envId" {
-  type        = string
-  description = "envId for PingOne Protect connector"
-}
-variable "davinci_connection_PingOne-0020-Protect_region" {
-  type        = string
-  description = "region for PingOne Protect connector"
-}
-variable "davinci_connection_PingOne_clientId" {
-  type        = string
-  description = "clientId for PingOne connector"
-}
-variable "davinci_connection_PingOne_clientSecret" {
-  type        = string
-  description = "clientSecret for PingOne connector"
-  sensitive   = true
-}
-variable "davinci_connection_PingOne_envId" {
-  type        = string
-  description = "envId for PingOne connector"
-}
-variable "davinci_connection_PingOne_region" {
-  type        = string
-  description = "region for PingOne connector"
-}
diff --git a/configuration/ping-export-module/versions.tf b/configuration/ping-export-module/versions.tf
deleted file mode 100644
index 42aa573..0000000
--- a/configuration/ping-export-module/versions.tf
+++ /dev/null
@@ -1,10 +0,0 @@
-terraform {
-  required_version = ">= 1.3"
-
-  required_providers {
-    pingone = {
-      source  = "pingidentity/pingone"
-      version = ">= 1.0.0"
-    }
-  }
-}
diff --git a/configuration/variables_davinci_export.tf b/configuration/variables_davinci_export.tf
deleted file mode 100644
index d9ae133..0000000
--- a/configuration/variables_davinci_export.tf
+++ /dev/null
@@ -1,54 +0,0 @@
-variable "pingone_environment_id" {
-  type        = string
-  description = "The PingOne environment ID to configure DaVinci resources in"
-
-  validation {
-    condition     = can(regex("^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$", var.pingone_environment_id))
-    error_message = "The PingOne Environment ID must be a valid PingOne resource ID (UUID format)."
-  }
-}
-
-# Connection Variables
-
-variable "davinci_connection_PingOne-0020-Protect_clientId" {
-  type        = string
-  description = "clientId for PingOne Protect connector"
-}
-
-variable "davinci_connection_PingOne-0020-Protect_clientSecret" {
-  type        = string
-  description = "clientSecret for PingOne Protect connector"
-  sensitive   = true
-}
-
-variable "davinci_connection_PingOne-0020-Protect_envId" {
-  type        = string
-  description = "envId for PingOne Protect connector"
-}
-
-variable "davinci_connection_PingOne-0020-Protect_region" {
-  type        = string
-  description = "region for PingOne Protect connector"
-}
-
-variable "davinci_connection_PingOne_clientId" {
-  type        = string
-  description = "clientId for PingOne connector"
-}
-
-variable "davinci_connection_PingOne_clientSecret" {
-  type        = string
-  description = "clientSecret for PingOne connector"
-  sensitive   = true
-}
-
-variable "davinci_connection_PingOne_envId" {
-  type        = string
-  description = "envId for PingOne connector"
-}
-
-variable "davinci_connection_PingOne_region" {
-  type        = string
-  description = "region for PingOne connector"
-}
-
